### PR TITLE
Bump strengthify

### LINKED ISSUE #11033: Strenghtify uses javascript eval
For https://github.com/orgs/nextcloud/projects/18

The strenghtify plugin we use for jquery toindicate password strength uses eval. This prevents a default stricter CSP.

We should look into how we can fix this properly.

### PR DESCRIPTION
**Pull request by rullzer**
Fixes #11033
For https://github.com/orgs/nextcloud/projects/18

Move to a stricter CSP safe strengthify

Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: core/vendor/strengthify/jquery.strengthify.js [hunk 1]
```diff
@@ -203,13 +207,11 @@
...
 
-                $.ajax({
-                    cache: true,
-                    url: options.zxcvbn
-                }).done(function(content) {
-                    eval(content);
-                    $elem.bind('keyup input change', drawSelf);
-                });
+                var script = document.createElement("script");
+                script.src = options.zxcvbn;
```
Comment (skjnldsv) [inline core/vendor/strengthify/jquery.strengthify.js]: Yes! :D 
Far better!

Comment (skjnldsv) [reply core/vendor/strengthify/jquery.strengthify.js]: But I thought we were going to need some kind of csp policy of some sort?

**Review by skjnldsv — APPROVED** [review]

File: core/vendor/strengthify/jquery.strengthify.js (hunk 1 above)
Comment (MorrisJobke) [reply core/vendor/strengthify/jquery.strengthify.js]: As of now it worked with the existing ones on my Apache setup. So this is fine from my side to get in.

**Pull request merged**